packages/components/src/lib/controllers/selection-controller.js

```diff
@@ -1,12 +1,25 @@
-import UIEventController from '@/lib/controllers/ui-event-controller.js'
+import UIEventController from '@comp/lib/controllers/ui-event-controller.js'
+import HTMLSelector from '@comp/lib/selection/media/html-selector.js'
 import { PsEvent } from 'alpheios-data-models'
 
 /**
  * `SelectionController` manages all aspects of text selection.
  */
 export default class SelectionController {
-  constructor () {
+  /**
+   * @param {Function} getDefaultLangCodeFn - A function that returns a default language code in ISO 639-3 format.
+   *        This language code will be used to determine a language if language info is not provided by the page.
+   */
+  constructor (getDefaultLangCodeFn = () => 'lat') {
```

Comment:
I didn't quitely understand, why do you use a function for defining default languageCode.
Default languageCode should be a constant by its sense.
I din't find in the code where it is used as function.

Reply:
Default language code is calculated via options as `return this.options.overridePreferredLanguage ? this.options.textLangCode : this.featureOptions.items.preferredLanguage.currentValue`. Because of this, it can change any time during the project lifetime. So we need it to be calculated dynamically, It can either be a getter or a function. I've decided to use a function because it serves as a good indication of a dynamic nature of the value.